src/HtmlUtils.js

```diff
@@ -136,6 +142,33 @@ var sanitizeHtmlParams = {
             attribs.rel = 'noopener'; // https://mathiasbynens.github.io/rel-noopener/
             return { tagName: tagName, attribs : attribs };
         },
+        '*': function(tagName, attribs) {
+            // Only allow certain CSS attributes to avoid XSS attacks
+            // Sanitizing values to avoid `url(...)` and `expression(...)` attacks
+            if (!attribs.style) {
+                return { tagName: tagName, attribs: attribs };
+            }
+
+            const pairs = attribs.style.split(';');
+            let sanitisedStyle = "";
+            for (let i = 0; i < pairs.length; i++) {
```

Comment:
hand-sanitizing CSS like this scares me quite a lot.  In this instance it feels quite fragile, and will fail if there's any white-space in the expression? which feels wrong, given whitespace is valid in CSS so we're defining our own custom derivative of CSS here which is generally bad karma.

However, in the longer term, we probably do want users to be able to enter CSS into their HTML messages in order to get pixel-perfect layout (and hopefully sandboxing tech will have caught up by then to ensure we can render the contents in some kind of sandbox to protect the rest of the app from it).  So i guess it's not that bad here... as long as the sanitizer is neither to lax nor too strict.